lib/rules/lint-no-meta-redirect-with-time-limit.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const AstNodeInfo = require('../helpers/ast-node-info');
+const Rule = require('./base');
+
+module.exports = class noMetaRedirectWithTimeLimit extends Rule {
+  logNode({ node, message }) {
+    return this.log({
+      message,
+      line: node.loc && node.loc.start.line,
+      column: node.loc && node.loc.start.column,
+      source: this.sourceForNode(node),
+    });
+  }
+  visitor() {
+    return {
+      ElementNode(node) {
+        if (AstNodeInfo.hasAttribute(node, 'hidden')) {
+          return;
+        }
+
+        if (AstNodeInfo.hasAttributeValue(node, 'aria-hidden', 'true')) {
+          return;
+        }
+
+        const isMeta = AstNodeInfo.isElementNode(node, 'meta');
+        const hasMetaRedirect = AstNodeInfo.hasAttribute(node, 'http-equiv');
+
+        if (isMeta && hasMetaRedirect) {
+          const contentAttr = AstNodeInfo.hasAttribute(node, 'content');
+          const contentAttrValue = AstNodeInfo.elementAttributeValue(node, 'content');
+          if (contentAttr) {
+            if (contentAttrValue.charAt(0) > 0) {
```

Comment:
I think it could be simplyfied to 
```suggestion
            if (!AstNodeInfo.hasAttributeValue(node, 'content', '0')) {
```